### DISCUSSION TITLE
Query

### DISCUSSION THREAD

**Discussion by estudiobarragan** [maize-tech/laravel-markable#18] [Q&A] [answered]
First, thaks a lot for this wonderful package. Second, a problem. A have table users and post. The users bookmarks post and i need to shows all de post who a particular user bookmark. So, Post::where('state',5)->and  here, how i cant know witch post are bookmars for this user?

**Comment by estudiobarragan** [comment, marked as answer]
I respond to my selfe (sorry...)
$this->posts = Post::where('state_id', 5)
      ->with(['user', 'categoria', 'tags'])
      ->**whereHasBookmark(auth()->user())** // this is the answer
      ->orderBy('publicar', 'desc')
      ->get();

**Reply by riccardodallavia** [reply]
That's correct!
